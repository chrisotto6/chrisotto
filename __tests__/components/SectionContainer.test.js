import { render } from '@testing-library/react'
import SectionContainer from '@/components/SectionContainer'

it('renders SectionContainer unchanged', () => {
  const { container } = render(<SectionContainer>{'test'}</SectionContainer>)
  expect(container).toMatchSnapshot()
})
