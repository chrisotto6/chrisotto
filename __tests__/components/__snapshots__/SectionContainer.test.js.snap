// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SectionContainer unchanged 1`] = `
<div>
  <div
    class="mx-auto max-w-3xl px-4 sm:px-6 xl:max-w-5xl xl:px-0"
  >
    test
  </div>
</div>
`;
